distance_attenuation/plot_calldata.py

```diff
@@ -0,0 +1,36 @@
+from distance_attenuation import *
+from IPython import embed
+from numpy import *
```

Comment:
let me take the freedom to comment :) 

```from xyz import *``` imports are sometimes problematic since, especially when importing large packages like numpy, you overwrite things accidentally. Further, the origin of the methods you are using is not that obvious